css/standalone/dashboard.scss

```diff
@@ -89,6 +89,7 @@ $break_tablet: 1400px;
         margin-bottom: -10px;
         z-index: 1;
         box-sizing: content-box;
+        height: 100px;
```

Comment:
is it required ? fixing the mini dashboard height where it should be calculated from it number of cells seems strange.

Reply:
Without we have a space at bottom of each cell 
![image](https://user-images.githubusercontent.com/7335054/187848068-8a976b21-6626-4130-9076-3966e3816d31.png)
and drag feature is completely break
perhaps there is another way ..

Reply:
to be 100% sure, transform the property to `min-height`.

Reply:
No change with ```min-height``` property 
![image](https://user-images.githubusercontent.com/7335054/187905701-5cea068c-1745-49b1-8802-108b61450442.png)